### PR TITLE
Enabling cd pipeline

### PR DESCRIPTION
**Pull request by taojang**
brings kontrolletti under team torch

### CODE REVIEW

**Comment by ChristianLohmann** [comment]
👍

File: conf/application.test.db.conf [hunk 1]
```diff
@@ -3,7 +3,7 @@ include "application"
 slick.dbs.default.driver="dao.KontrollettiPostgresDriver$"
 slick.dbs.default.db.driver=org.postgresql.Driver
 slick.dbs.default.db.url="jdbc:postgresql://localhost:5432/kontrolletti_test?characterEncoding=UTF-8"
-slick.dbs.default.db.url=${?DATABASE_URL_TEST}
+slick.dbs.default.db.url="jdbc:postgresql:"${?DB_SUBNAME}"?characterEncoding=UTF-8"
```
Comment (harti2006) [inline conf/application.test.db.conf]: why is the key "slick.dbs.default.db.url" duplicated?

**Comment by taojang** [comment]
:+1:

File: conf/application.test.db.conf (hunk 1 above)
Comment (taojang) [inline conf/application.test.db.conf]: This is a feature from the [Typesafe Config library](https://github.com/typesafehub/config#optional-system-or-env-variable-overrides)
It allows to use env var and if it does not exists, fallback with previous value

**Pull request merged**